frontend/public/components/nav/confirmNavUnpinModal.tsx

```diff
@@ -17,19 +19,22 @@ const confirmNavUnpinModal = (resource: string, pinnedResources: string[], updat
   const label = modelFor(resource)?.labelPlural;
   const title = (
     <>
-      <YellowExclamationTriangleIcon className="co-icon-space-r" /> Remove from navigation?
+      <YellowExclamationTriangleIcon className="co-icon-space-r" />{' '}
+      {i18next.t('public~Remove from navigation?')}
     </>
   );
   const message = (
     <span>
-      Are you sure you want to remove <strong>{label}</strong> from navigation?
+      <Trans i18nKey="public~Are you sure you want to remove from navigation?">
```

Comment:
In public.json,
"Are you sure you want to remove from navigation?": "Are you sure you want to remove <1>{{label}}</1> from navigation?",

It is a const type message.
  const message = (
    <span>
      <Trans i18nKey="public~Are you sure you want to remove from navigation?">
        Are you sure you want to remove <strong>{{ label }}</strong> from navigation?
      </Trans>
    </span>
  );

Reply:
The translators won't be able to translate what's in the Trans component because they will only see the i18nKey. Can you please remove the i18nKey? Thanks!

Reply:
Yes, removed i18nKey.
<Trans ns="public">
        Are you sure you want to remove <strong>{{ label }}</strong> from navigation?
      </Trans>